src/distances/bures.jl

```diff
@@ -0,0 +1,74 @@
+# Code from @devmotion
+# https://github.com/devmotion/\
+# CalibrationErrorsDistributions.jl/blob/main/src/distances/bures.jl
+
+"""
+    tr_sqrt(A::AbstractMatrix)
+
+Compute ``\\operatorname{tr}\\Big({\\big(A^{1/2} B A^{1/2}\\big)}^{1/2}\\Big)``.
```

Comment:
The docstring is not correct anymore:
```suggestion
Compute ``\\operatorname{tr}\\big(A^{1/2}\\big)``.
```